<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GerenciadorObjetosDialog</class>
 <widget class="QDialog" name="GerenciadorObjetosDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerador de Formas</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>258</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnExcluirForma">
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnModificarForma">
      <property name="text">
       <string>Modificar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDesenhar">
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lblForma">
      <property name="text">
       <string>Forma:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboFormas"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>117</width>
     <height>173</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="lblCoordenadaX1">
      <property name="text">
       <string>Coordenada X1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinX1">
      <property name="maximum">
       <double>600.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCoordenadaX2">
      <property name="text">
       <string>Coordenada X2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinX2">
      <property name="maximum">
       <double>600.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCoordenadaX3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Coordenada X3:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinX3">
      <property name="maximum">
       <double>600.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>118</width>
     <height>173</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="lblCoordenadaY1">
      <property name="text">
       <string>Coodernada Y1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinY1">
      <property name="maximum">
       <double>600.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCoordenadaY2">
      <property name="text">
       <string>Coodernada Y2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinY2">
      <property name="maximum">
       <double>600.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblCoordenadaY3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Coordenada Y3:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinY3">
      <property name="maximum">
       <double>600.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>117</width>
     <height>173</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="lblCoordenadaZ1">
      <property name="text">
       <string>Coordenada Z1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinZ1"/>
    </item>
    <item>
     <widget class="QLabel" name="lblCoordenadaZ2">
      <property name="text">
       <string>Coordenada Z2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinZ2"/>
    </item>
    <item>
     <widget class="QLabel" name="lblCoordenadaZ3">
      <property name="text">
       <string>Coordenada Z3:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinZ3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>123</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetFixedSize</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="lblRaio">
        <property name="text">
         <string>Raio:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spinRaio">
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="lblTamanho">
        <property name="text">
         <string>Tamanho</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spinTamanho"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnCor">
        <property name="text">
         <string>Selecionar Cor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GerenciadorObjetosDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GerenciadorObjetosDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
